frontend/app/webpack.common.config.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
       inject: 'body',
       filename: 'index.html'
     }),
-    new webpack.NoEmitOnErrorsPlugin()
+    new webpack.NoEmitOnErrorsPlugin(),
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
```

Comment:
Looks like now that you've ignored all moment locales, the resulting strings are english (for example in the previous queries list). Can you have a look at it?

Reply:
Yes, my bad. We should use the ContextReplacementPlugin instead: https://webpack.js.org/plugins/context-replacement-plugin/

Example:
```
new webpack.ContextReplacementPlugin(
  /moment[\/\\]locale$/, /de|fr|hu/
)
```